### DISCUSSION TITLE
Cura 5.0 is officially released, JPEG Thumbnail doesn't work

### DISCUSSION THREAD

**Discussion by jvcjbl** [mriscoc/Ender3V2S1#160] [Q&A] [answered]
Now that Cura 5.0 is officially released I want to start messing around with it since I use PrusaSlicer exclusively. For the life of me I can't get the thumbnail script to work. Cura 5.0 just doesn't even see the script in the scripts folder. Anyone running the official release have the thumbnail script working?

**Comment by SorenRR** [comment, marked as answer]
Have you tried this ??
[https://github.com/mriscoc/Ender3V2S1/discussions/139#discussioncomment-2670926](https://github.com/mriscoc/Ender3V2S1/discussions/139#discussioncomment-2670926)

**Reply by jvcjbl** [reply]
I was going to, but I've seen several people say they've implemented it and it doesn't work. I wanted to wait for either confirmation from someone that it works or Miguel saying it does before I do it. I'm actually doing my first print right now using Cura 5 and for whatever reason the remaining print time isn't working.
